lib/sequelize.js

```diff
@@ -228,6 +232,12 @@ class Sequelize {
 
     this.dialect = new Dialect(this);
     this.dialect.QueryGenerator.typeValidation = options.typeValidation;
+    if (this.options.operatorsAliases === true) {
+      Utils.deprecate('String based operators are now deprecated. Please use Symbol based operators for better security, read more at http://docs.sequelizejs.com/manual/tutorial/querying.html#operators');
```

Comment:
One second thought, I don't think we should remove complete support for operator aliases.

Simply put, I still want to be able to pass in a JSON object from an HTTP request into the `where` property.

Reply:
I think we should disable them by default in the future, but should not completely remove them, there are definitely use-cases for that, I think

Reply:
Yea I agree.
I don't think there is any thought of deprecating them but users should define the ones they want to use and not have all of them available by default.

In the future it might be useful to define aliases in the query options. So for example if you have one query that you know you gonna pass right through and you want to allow for 'like' operator you can just add an alias for it that it will only be available for that specific query